rmgpy/rmg/model.py

```diff
@@ -1257,11 +1257,36 @@ def addSeedMechanismToCore(self, seedMechanism, react=False):
 
         seedMechanism = database.kinetics.libraries[seedMechanism]
 
-        for entry in seedMechanism.entries.values():
-            rxn = LibraryReaction(reactants=entry.item.reactants[:], products=entry.item.products[:],\
-             library=seedMechanism.label, specificCollider=entry.item.specificCollider, kinetics=entry.data, duplicate=entry.item.duplicate,\
-             reversible=entry.item.reversible
-             )
+        for entry in seedMechanism.entries.values():                
+            if entry._longDesc and 'Originally from reaction library: ' in entry._longDesc:
```

Comment:
Could you combine this commit with the commit moving the new method to 'family.py'?

Also, I am not sure what 'no change in pdep handling' indicates in the commit message. When merging, could you reword it to describe that loading does not work properly with jobs that output pdep reactions (and maybe why they don't work properly). This would be super helpful for future development.